src/audio/wav64.c

```diff
@@ -175,12 +192,14 @@ static void waveform_vadpcm_read(void *ctx, samplebuffer_t *sbuf, int wpos, int
 	if (seeking) {
 		if (wpos == 0) {
 			memset(&vhead->state, 0, sizeof(vhead->state));
-			vhead->current_rom_addr = wav->rom_addr;
+			vhead->currentFile = wav->currentFile; // TODO: The file* is not is not an instance, so seeking may not be safe but it works.
+			lseek(vhead->currentFile, wav->baseOffset, SEEK_SET);
```

Comment:
I don't understand the comment. Can you explain?

Reply:
What I meant is: The original assignment of rom_addr could be used to allow wav->rom_addr and vhead->current_rom_addr to diverge later on. This is currently not happening based on testing, but if it were to happen we woudl need to open the file again here.